WoWDatabaseEditor.Common/WDE.Common/Providers/IDataDefinitionsProvider.cs

```diff
@@ -0,0 +1,13 @@
+﻿using System.Collections.Generic;
+using WDE.Module.Attributes;
+using WDE.Common.Managers;
+
+namespace WDE.Common.Providers
+{
+    [NonUniqueProvider]
+    public interface IDataDefinitionsProvider
+    {
+        string GetDataCategoryName();
+        IEnumerable<IDataDefinitionPresenter> GetDataDefinitionPresenters();
```

Comment:
Can we change name? Presenter sound too much like „view”. GetDataDefinitionEditors?